tests/test_adcirc_mesh.py

```diff
@@ -168,3 +168,17 @@ def test_make_plot_flat_domain(nodes, elements, boundaries, mocker):
 
     mocker.patch('matplotlib.pyplot.show')
     mesh.make_plot()
+
+
+def test_mesh_equality(nodes, elements, boundaries, wet_nodes, wet_elements):
+    mesh_1 = AdcircMesh(nodes, elements, boundaries)
+    mesh_2 = AdcircMesh(nodes, elements, boundaries)
+    mesh_3 = mesh_1
```

Comment:
shouldn't this be a `copy()` or a `deepcopy()`?

Reply:
updated